### PR TITLE
Implement the `--keep` option for `tmt clean guests`

### PR DESCRIPTION
**Pull request by skycastlelily**
Pull Request Checklist

* [x] implement the feature
* [ ] write the documentation
* [x] extend the test coverage
* [ ] update the specification
* [ ] adjust plugin docstring
* [ ] modify the json schema
* [ ] mention the version
* [x] include a release note

### CODE REVIEW

File: docs/releases.rst [hunk 1]
```diff
@@ -20,6 +20,10 @@ environment files are found.
 The ``tmt try`` command now supports the new
 :ref:`/stories/cli/try/option/arch` option.
 
+Add support for ``--keep`` params for the ``clean_guests`` subcommand.
```
Comment (therazix) [inline docs/releases.rst:23]: This should now be under 1.40

**Review by therazix — APPROVED** [review]

File: docs/releases.rst (hunk 1 above)
Comment (skycastlelily) [reply docs/releases.rst]: Thanks for your review: https://github.com/teemtee/tmt/pull/3182/commits/50cf34da8accf67d1d3de0a96b02cd88ea43fc69

File: docs/releases.rst [hunk 2]
```diff
@@ -5,6 +5,14 @@
 ======================
 
 
+tmt-1.40.0
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Add support for ``--keep`` params for the ``clean_guests`` subcommand.
```
Comment (psss) [inline docs/releases.rst:11]:
```suggestion
Add support for ``--keep`` params for the ``tmt clean`` subcommand.
```

Better to use the actual subcommand in the form which users actualy see.

**Review by psss — APPROVED** [review]
Thanks for improving this!

File: docs/releases.rst (hunk 2 above)
Comment (psss) [reply docs/releases.rst]: Updated during the rebase.

**Pull request merged**